registration.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE HTML>
+<!--
+	Editorial by HTML5 UP
+	html5up.net | @ajlkn
+	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
+-->
+<html>
+	<head>
+		<title>MPI-SHH Summer School!</title>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+		<link rel="stylesheet" href="assets/css/main.css" />
+		<link rel="stylesheet" href="assets/css/custom.css" />
+		<script src="https://kit.fontawesome.com/669895648b.js" crossorigin="anonymous"></script>
+	<!-- <script defer src="assets/fontawesome-free-5.15.2-web/js/all.js"></script> -->
+	</head>
+	<body class="is-preload">
+
+		<!-- Wrapper -->
+			<div id="wrapper">
+
+				<!-- Main -->
+					<div id="main">
+						<div class="inner">
+
+							<!-- Header -->
+ 								<header id="header">
+									<a href="index.html" class="logo"><img src='images/test.png'></img>MPI-SHH Summer School</a>
+									<!-- <ul class="icons">
+										<li><a href="#" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
+									</ul> -->
+								</header>
+                                
+                                <section id='introduction'>
+                                    <header class="major">
+										<h2>About MPI-SHH Summer School</h2>
```

Comment:
I'd propose to write just "About". It should be clear that it's about the summer school.